package.json

```diff
@@ -24,10 +24,13 @@
     "test": "yarn sdk:test",
     "format": "yarn sdk:prettier",
     "protocol:generate": "yarn workspace @beanstalk/protocol generate",
+    "jest:clearcache": "jest --clearCache",
     "sdk:generate": "yarn workspace @beanstalk/sdk generate",
     "sdk:dev": "yarn workspace @beanstalk/sdk dev",
     "sdk:build": "yarn workspace @beanstalk/sdk build",
     "sdk:test": "jest --selectProjects sdk --silent --runInBand --",
+    "sdk-wells:test": "jest --selectProjects sdk-wells --runInBand --silent=false --",
```

Comment:
I'd like to move away from having root level scripts like this. Can we move these to the project package.json and when we need to invoke, we can do it with `yarn workspace sdk-well test` ?

Also, if we make use of the `:` thing, which makes commands globally accessible anywhere in the workspace, you could have this in the project/packagejson/scripts:
- test: 'whatever'
- sdk-well:test: 'yarn test'

Then from anywhere, you could do `yarn sdk-well:test` and it will work.

this is just an fyi, not really sure we need to do this piece

Reply:
I want to talk more about this. I realized right now the package name is `@beanstalk/wells` and maybe this doesn't make the most sense.

Reply:
Also the jest config is in the root directory :-/ so that could cause issues